doc/api/buffer.md

```diff
@@ -203,7 +203,20 @@ The character encodings currently supported by Node.js include:
 
 * `'binary'`: Alias for `'latin1'`.
 
-* `'hex'`: Encode each byte as two hexadecimal characters.
+* `'hex'`: Encode each byte as two hexadecimal characters. Data truncation
+  may occur for unsanitized input. For example:
+
+```js
+Buffer.from('1ag', 'hex');
+// Prints <Buffer 1a>, data truncated when first non-hexadecimal value
+// ('g') encountered
+
+Buffer.from('1a7g', 'hex');
+// Prints <Buffer 1a>, data truncated when data ends in single digit ('7').
+
+Buffer.from('1634', 'hex');
+// Prints <Buffer 16 34>, fully qualified hexadecimal data
```

Comment:
Optional nit: Saying that the data was not truncated might be easier to understand than describing it as fully-qualified.

```suggestion
// Prints <Buffer 16 34>; all data represented.
```

Reply:
@Trott, thanks. Fixed it.